style.css

```diff
@@ -0,0 +1,62 @@
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+}
+
+#img1 {
+  justify-content: flex-start;
+}
+
+.ic_icon {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  flex: 0;
+  padding: 8%;
+}
+
+#btn {
+  justify-content: space-between;
+  border-radius: 8%;
+  background: #1a2236;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.group48 {
+  display: flex;
+  flex-direction: column;
+}
+
+#title {
+  font-family: 'Courier New', Courier, monospace; /* Playfair Displa; */
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 50px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+#text {
+  font-family: 'Courier New', Courier, monospace;
```

Comment:
- [ ] Kindly use the same `font-family` (Poppins) for all the text in your portfolio to follow the project requirement and show your ability to understand how to parse a Figma design to create a UI.